prow/plugins/lifecycle/close.go

```diff
@@ -101,9 +105,19 @@ func handleClose(gc closeClient, log *logrus.Entry, e *github.GenericCommentEven
 	}
 
 	log.Info("Closing issue.")
-	if err := gc.CloseIssue(org, repo, number); err != nil {
-		return fmt.Errorf("Error closing issue: %w", err)
+	var reply string
+	if closeNotPlannedRe.MatchString(e.Body) {
+		if err := gc.CloseIssueAsNotPlanned(org, repo, number); err != nil {
+			return fmt.Errorf("Error closing issue: %w", err)
```

Comment:
nit: can we elaborate the error message to be `Error closing issue as not-planned`?